modules/nw-ingress-setting-thread-count.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+[id="nw-ingress-setting-thread-count_{context}"]
+= Setting Ingress Controller thread count
+A cluster administrator can  increase the amount of incoming connections your cluster can handle by setting the thread count. This can be accomplished in a couple of different ways.
+
+You can patch an existing Ingress Controller to increase the amount of Ingress Controller threads.
+
+. Procedure
+The following assumes that you have already created an Ingress Controller.
+
+.. Update the Ingress Controller to increase the number of threads
++
+[source,terminal]
+----
+$ oc patch ingresscontroller/default --type=merge -p '{"spec": {"threading": {"count": <new-thread-count>} } }'
```

Comment:
example to increase the number of threads to 8:
`oc -n openshift-ingress-operator patch ingresscontroller/default --type=merge -p '{"spec":{"tuningOptions": {"threadCount": 8}}}'`